dothething/dothething/ViewModels/ClipsViewModel.swift

```diff
@@ -244,9 +246,9 @@ class ClipsViewModel: ObservableObject, ImagePickerMessenger {
                             let thumbnail = Thinger.getThumbnail(url: tempFileUrl, degreesToRotate: self.videoDegreesToRotate)
                             // highlight first clip
                             if self.clips.count == 0 {
-                                self.clips.append(Clip(url: tempFileUrl, thumbnail: thumbnail, isHighlighted: true, metadata: clip))
+                                self.clips.append(Clip(url: tempFileUrl, thumbnail: thumbnail, isHighlighted: true, metadata: clip, showCode: false))
                             } else {
-                                self.clips.append(Clip(url: tempFileUrl, thumbnail: thumbnail, isHighlighted: false, metadata: clip))
+                                self.clips.append(Clip(url: tempFileUrl, thumbnail: thumbnail, isHighlighted: false, metadata: clip, showCode: false))
```

Comment:
The changes to the initialization of `Clip` instances within the `ClipsViewModel` are also consistent with the previous changes, ensuring that the `showCode` property is set correctly for each clip.